### PR TITLE
Daydream README enhancement

### LINKED ISSUE #12: Create walkthrough example of how to use daydream

### PR DESCRIPTION
**Pull request by Cleop**
This is a first edit which could now benefit from some feedback for future improvement.

Fixes #12, #1 

- Creates walkthrough of how to use daydream with an example (featuring gif and screenshots)
- Creates to do list of things to add to the README
- Adds resource about use of nightmare for web scraping from #1 
- Creates context for tutorial (referring to need for node and nightmare installation)*
- Adds psychological reasoning behind inaccuracy of manual testing

\* which may be added to at a later date to help explain how to install node/nightmare but is currently being left out as the context of this tutorial is for non-technical people / a client who may wish to build tests on an already existing project. It is not trying to teach them how to create a project from scratch in this scope!

### CODE REVIEW

**Comment by nelsonic** [comment]
@Cleop thanks for assigning me to the PR. is it OK if I _unassign_ you for clarity?

File: README.md [hunk 1]
```diff
@@ -8,47 +8,46 @@ you discover that you've got "***web super powers***"!!
...
+  + GitHub: https://github.com/segmentio/nightmare
 
 #### Not Covered
 
-_Awkwardly_, Google _recently_ decided
-to call their Virtual Reality (VR) project "_Daydream_" ...
-see: [google.com/**daydream**](https://vr.google.com/daydream) <br />
+_Awkwardly_, Google recently decided
+to call their Virtual Reality (VR project "_Daydream_" ...
+see [google.com/**daydream**](https://vr.ogle.com/daydream),
```
Comment (nelsonic) [inline README.md]: https://vr.ogle.com/daydream times out ... 
image

is this a typo?

File: README.md [hunk 2]
```diff
@@ -70,21 +69,106 @@ of 30 mins of learning "***web super powers***",
...
 
 ## _How_?
 
 The _reason_ we have organized the tutorial with Daydream _first_
 is that you don't _need_ to know Nightmare.js to use Daydream,
 and it can be used by people without "coding" experience.
 
+### Daydream walkthrough
+
+**This tutorial assumes that you have node installed and [Nightmare installed](https://github.com/segmentio/nightmare) as a dependency**
```
Comment (nelsonic) [inline README.md]: I'd be _specific_ here and write `node.js` not just "node". 
(_non-technical people or complete beginners might not know that node & node.js are used interchangeably so worth being precise_)

**Comment by Cleop** [comment]
Thanks for reviewing @nelsonic and for the good spots. Review changes have been made 😄  please check again and merge if everything looks good!

File: README.md [hunk 3]
```diff
@@ -70,21 +69,106 @@ of 30 mins of learning "***web super powers***",
...
+
+Daydream does the hard work of recording our actions into a Nightmare test. However, we still need to add to the code it gives us for our test to work. Let's understand what Daydream has done for us:
+
+**1.** Set up your code to run Nightmare tests
+
+**2.** Written the code that tracks the user's journey throughout the code (ie. going to a given url, clicking on a button)
+
+<div style="text-align:center"><img src ="https://cloud.githubusercontent.com/assets/16775804/24255398/4d9567a8-0fdd-11e7-9888-0d33f330adf0.png" height="300px"/></div>
+
+If you're non-technical try making sense of the commands in part **2** (on the screenshot). They represent common user actions such as clicking on a page or taking a screenshot. The text within the parentheses indicates what these commands apply to ie. what html/css element is being clicked on.
```
Comment (nelsonic) [inline README.md:138]: note: we tend to use "semantic" line breaks to avoid having super-long-lines of text.

**Review by nelsonic — APPROVED** [review]
@Cleop this PR looks _really_ good. I've made a couple of comments.

**Pull request merged**